book_class.rb

```diff
@@ -0,0 +1,15 @@
+require './rental'
+require './person_class'
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(date, person)
+    Rental.new(date, person, self)
+  end
```

Comment:
Great on implementing the `many_to_many` and also `many_through` relationship between `person` and `book` :+1: